walletnode/services/senseregister/task.go

```diff
@@ -162,19 +153,24 @@ func (task *Task) run(ctx context.Context) error {
 	task.UpdateStatus(StatusTicketRegistered)
```

Comment:
should move 
```
task.UpdateStatus(StatusTicketAccepted)
	log.WithContext(ctx).Debug("close connections to supernodes")
	close(nodesDone)
	for i := range task.nodes {
		if err := task.nodes[i].Connection.Close(); err != nil {
			log.WithContext(ctx).WithFields(log.Fields{
				"pastelId": task.nodes[i].PastelID(),
				"addr":     task.nodes[i].String(),
			}).WithError(err).Errorf("close supernode connection failed")
		}
	}
	// new context because the old context already cancelled
	newCtx := context.Background()
	if err := task.waitTxidValid(newCtx, task.regSenseTxid, int64(task.config.RegArtTxMinConfirmations), 15*time.Second); err != nil {
		return errors.Errorf("wait reg-nft ticket valid: %w", err)
	}
```
to after calling task.nodes.UploadActionAct() successful

Reply:
resolved in next commit. pls review again.